addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -51,7 +51,7 @@
      */
     private static final String MESSAGE_PREFIX = "ascanrules.pathtraversal.";
 
-    private static final String NON_EXISTANT_FILENAME = "thishouldnotexistandhopefullyitwillnot";
+    private static final String NON_EXISTANT_FILENAME = "thishouldnonexistentandhopefullyitwillnot";
```

Comment:
For reference, there are a number of errors in the original line, although the fix was wrong.

```
private static final String NONEXISTENT_FILENAME = "ThisShouldNotExistAndHopefullyItWillNot";
```